ext/pcre/pcre2lib/pcre2_jit_compile.c

```diff
@@ -13742,7 +13742,7 @@ if (SLJIT_UNLIKELY(sljit_get_compiler_error(compiler)))
   sljit_free_compiler(compiler);
   SLJIT_FREE(common->optimized_cbracket, allocator_data);
   SLJIT_FREE(common->private_data_ptrs, allocator_data);
-  PRIV(jit_free_rodata)(common->read_only_data_head, compiler->allocator_data);
+  PRIV(jit_free_rodata)(common->read_only_data_head, allocator_data);
```

Comment:
I suppose, this and following changes are unrelated.

Reply:
Yes.... rebase issue.... ignore this